bucket/windterm.json

```diff
@@ -29,6 +23,14 @@
             "WindTerm"
         ]
     ],
+    "post_install": [
+        "info '[Portable Mode] Configuring profiles directory...'",
+        "'{ \"path\" : \".\" }' | Set-Content -Path \"$dir\\profiles.config\"",
```

Comment:
You should also check for existence of this file

Reply:
Normally is the file created by the application whenever a new version of the application is started for the first time. The file is located next to the application files. Each version of the application is located in its own directory. There is no need to test the file for existence, it cannot exist after installation unless it is part of the installation package (which is not).